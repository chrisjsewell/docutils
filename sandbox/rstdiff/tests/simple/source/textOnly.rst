.. role:: change-replaced
.. role:: change-replacement

.. class:: change-added

   An important addition.
   
This is some text distributed in a couple of lines. Let's count to
:change-replaced:`make things longer:` :change-replacement:`fill three lines:` one two three four five six seven eight nine ten
eleven twelve.

.. class:: change-deleted
   
   Some obsolete add-on.

