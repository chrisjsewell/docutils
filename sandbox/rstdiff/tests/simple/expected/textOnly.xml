<?xml version="1.0" encoding="utf-8"?>
<!-- Generated by Docutils 0.6 -->
<document source="input/textOnly.old.rst =&gt; input/textOnly.new.rst">
    <paragraph classes="change-added">
        An important addition.
    </paragraph>
    <paragraph>
        This is some text distributed in a couple of lines. Let's count to

        <inline classes="change-replaced">
            make things longer:
        </inline>
         
        <inline classes="change-replacement">
            fill three lines:
        </inline>
         one two three four five six seven eight nine ten
eleven twelve.
    </paragraph>
    <paragraph classes="change-deleted">
        Some obsolete add-on.
    </paragraph>
</document>
